thirdparty/clamav-db/zimbra-clamav-db/debian/zimbra-clamav-db.postinst

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 if [ "$1" = configure ]; then
-  /bin/chown -R zimbra:zimbra /opt/zimbra/data/clamav
-  /bin/chmod 644 /opt/zimbra/data/clamav/init/*.init
+	/bin/chown -R zimbra:zimbra /opt/zimbra/data/clamav
+	/bin/chmod 644 /opt/zimbra/data/clamav/init/*.init
+	if [ -f /opt/zimbra/data/clamav/db/daily.cvd ]; then
```

Comment:
why are we removing these files?

Reply:
We need to remove these to copy updated db files.
https://github.com/Zimbra/zm-core-utils/blob/develop/src/bin/zmclamdctl#L80